test/zdtm/static/cgroupv2_00.c

```diff
@@ -0,0 +1,131 @@
+#include <sys/mount.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "zdtmtst.h"
+
+const char *test_doc = "Check that some cgroup-v2 properties in kernel controllers are preserved";
+const char *test_author = "Bui Quang Minh <minhquangbui99@gmail.com>";
+
+char *dirname;
+TEST_OPTION(dirname, string, "cgroup-v2 directory name", 1);
+const char *cgname = "subcg00";
+
+int write_value(const char *path, const char *value)
```

Comment:
Let's move `write_value`/`read_value` helpers to `zdtm/lib`. As far as you adding two usage cases for it + as far as I can see we already have analogical helpers in `cgroup04` test.